src/url.js

```diff
@@ -527,3 +528,96 @@ export function checkCorsUrl(url) {
 export function tryDecodeUriComponent(component, opt_fallback) {
   return tryDecodeUriComponent_(component, opt_fallback);
 }
+
+/**
+ * If (tagName, attrName) is a CDN-rewritable URL attribute, returns the
```

Comment:
I think these additions are too specialized for `src/url.js` (even if it already is a hodgepodge).

Reply:
+1. The function names make sense in the context of a file like "sanitizer.js", but less so in a generic util class for URLs.